rust-plugins/react/index.js

```diff
@@ -119,4 +119,4 @@ switch (platform) {
     throw new Error(`Unsupported OS: ${platform}, architecture: ${arch}`);
 }
 
-export default binPath;
+export default (options)=>[binPath,options];
```

Comment:
This would introduce break change when using the new `@farmfe/plugin-react` in old `@farmfe/core`, a new package entry should be added to avoid break change